src/services/id-service.api-service.ts

```diff
@@ -36,4 +39,34 @@ export class IdServiceApiService {
         }
       });
   }
+
+  public async getByUri<T>(uri: string, getSso: () => Promise<string>): Promise<IHttpResponse<T>> {
+    try {
+      const sso = await getSso();
+      const response = await this.http.createRequest(`${oeAppConfig.idServiceUrl + '/uris?uri=' + uri}`)
+      .asGet()
+      .withHeader('Authorization', 'Bearer ' + sso)
+      .send() as IHttpResponse<IIdServiceResponse>; 
+
+      if (response.content.location) {
+        try {
+          const locationResponse = await this.http.createRequest(response.content.location).asGet()
+          .withHeader('Authorization', 'Bearer ' + sso)
+          .send() as IHttpResponse<T>;
+
+          const content = locationResponse.content;
+          const etag = locationResponse.headers.get('ETag');
```

Comment:
same here: is deze nog ok als er geen etags zijn?

Reply:
ja, dan is dat gewoon null/undefined zoals vroeger